### PR TITLE
Pill color for correct & incorrect answers

### PR DESCRIPTION
**Pull request by nedredmond**
## Summary
Match marker color for correct & incorrect answers.
Also disables interaction for correct answers, matching marker behavior.

Isolates component and adds interactive story doc.

Issue: LC-1498

Screenshot 2023-11-29 at 10 28 25 AM

https://github.com/Khan/perseus/assets/23404711/64157cdc-ed97-4546-89e0-a5382ab3d2a5

## Testing Strategy:
Fire up storybook and play in the new story

### CODE REVIEW

File: packages/perseus/src/widgets/label-image/answer-pill.tsx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    );
+};
+
+const styles = StyleSheet.create({
+    bringToFront: {
+        boxShadow: `0 8px 8px ${Color.offBlack64}`,
+        zIndex: 1000,
+    },
+    correct: {
+        backgroundColor: "#00880b", // WB green darkened by 18%
```
Comment (jeremywiebe) [inline packages/perseus/src/widgets/label-image/answer-pill.tsx:88]: **suggestion**: There is a WB function `fade()` that might encode this more clearly: https://github.com/Khan/wonder-blocks/blob/main/packages/wonder-blocks-color/src/util/utils.ts#L75

I think this colour could then be: 

```suggestion
        backgroundColor: fade(Color.green, 0.18),
```

**Review by jeremywiebe — APPROVED** [review]
Nice. Thanks for the test coverage and new stories!

File: packages/perseus/src/widgets/label-image/answer-pill.tsx (hunk 1 above)
Comment (nedredmond) [reply packages/perseus/src/widgets/label-image/answer-pill.tsx:88]: @jeremywiebe `fade` does the opposite-- it adds transparency to the color. Maybe I could use `mix` with black instead?

Comment (jeremywiebe) [reply packages/perseus/src/widgets/label-image/answer-pill.tsx:88]: Oops, sorry for the rabbit hole. :)

**Pull request merged**